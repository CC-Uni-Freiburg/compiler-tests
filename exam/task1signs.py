# Author: Alexander Pfefferle
#in=3661339
#in=3316
#in=-2323257
#in=-914
#golden=12141000124-3346463846-770392021221234568981104-4005-7002541475769-2057-783
a=input_int()
b=input_int()
c=input_int()
d=input_int()
print(a*b)
print(a*d)
print(c*b)
print(c*d)
print(a//b)
print(a//d)
print(c//b)
print(c//d)
print(a%b)
print(a%d)
print(c%b)
print(c%d)
