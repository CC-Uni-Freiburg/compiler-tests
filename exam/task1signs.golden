12141000124-3346463846-770392021221234568981104-4005-7002541475769-2057-783
